src/averell/readers/disco.py

```diff
@@ -1,5 +1,65 @@
+import xml.etree.ElementTree as ETree
+from pathlib import Path
+
+
+def parse_xml(xml_file):
+    """
+    XML TEI poem parser for 'disco' corpus
+    :param xml_file: path of xml file
```

Comment:
A bit more descriptive please, also use capital letters! and try to define the type of variable, i.e. 'Path for the xml file'

Reply:
Done